Tools/ardupilotwaf/px_uploader.py

```diff
@@ -667,6 +668,9 @@ def main():
                         # OS X, don't open Windows and Linux ports
                         if "COM" not in port and "ACM" not in port:
                             up = uploader(port, args.baud_bootloader, baud_flightstack, args.baud_bootloader_flash)
+                    elif "cygwin" in _platform:
```

Comment:
since it matches linux, why not make it:
`if "linux" in _platform or "cygwin" in _platform
`